packages/trimerge-sync/src/TrimergeClient_Remote.test.ts

```diff
@@ -145,46 +145,32 @@ Array [
     "remoteSave": "ready",
   },
   Object {
-    "localRead": "loading",
```

Comment:
These delta's are because we're not immediately reporting success when saving any more.